restcomm/restcomm.telephony/src/main/java/org/restcomm/connect/telephony/CallManager.java

```diff
@@ -649,6 +653,134 @@ private void info(final SipServletRequest request) throws IOException {
         }
     }
 
+    private void transfer(SipServletRequest request) throws Exception {
```

Comment:
@vmykhailiuk there is no response to the incoming Refer request and as a result there are several retransmitions of the REFER request and later when session expires there is an expection also.

As we have the Call actor, the IncomingPhoneNumber and the Refer url it will be safe to send 202 Accepted and Refer Notify 100 Trying.